force-app/main/default/classes/TriggerHelper.cls

```diff
@@ -0,0 +1,3 @@
+public class TriggerHelper {
+    public static Boolean isTriggerRunning = false;
+}
```

Comment:
Try to see if you can add this one liner to the TriggerHandler that you are already using